src/main/drivers/accgyro/accgyro_spi_mpu9250.c

```diff
@@ -143,7 +143,16 @@ static void mpu9250AccAndGyroInit(gyroDev_t *gyro) {
     mpu9250SpiWriteRegisterVerify(&gyro->bus, MPU_RA_PWR_MGMT_1, INV_CLK_PLL);
 
     //Fchoice_b defaults to 00 which makes fchoice 11
-    const uint8_t raGyroConfigData = gyro->gyroRateKHz > GYRO_RATE_8_kHz ? (INV_FSR_2000DPS << 3 | FCB_3600_32) : (INV_FSR_2000DPS << 3 | FCB_DISABLED);
+    uint8_t raGyroConfigData = gyro->gyroRateKHz = INV_FSR_2000DPS << 3;
```

Comment:
Erm, this is the same double assignment that broke ICM20689 (see #4842).

Reply:
Thanks. Fixed.

Reply:
#4842 led to me spending time on Christmas Day bisecting Betaflight - don't want to repeat that on New Year's day. :grinning: